src/wallet/wallet.cpp

```diff
@@ -251,11 +259,18 @@ bool CWallet::AddWatchOnly(const CScript &dest)
 {
```

Comment:
Grr, meant for this to be on AddWatchOnly at line https://github.com/bitcoin/bitcoin/pull/9108/files#diff-b2bb174788c7409b671c46ccc86034bdL251